ui/pages/Project/reducers.js

```diff
@@ -145,6 +145,34 @@ export const updateLocusLists = (values) => {
   }
 }
 
+export const loadUserOptions = () => {
```

Comment:
this is actually a weird merge artifact, this function got moved to `redux/rootReducers` and then you ended up adding it back here by accident but its not longer needed here